docs/contributing/zulipbot-usage.md

```diff
@@ -85,12 +85,6 @@ assignee(s) if they are still working on the issue.
     issue's current assignee(s) and the "in progress" label to allow others to
     work on an inactive issue.
 
```

Comment:
After this is added in zulipbot we can replace this with Circle CI.

Reply:
Opened https://github.com/zulip/zulipbot/issues/172 to track this.